src/types/line.js

```diff
@@ -107,9 +107,11 @@ LineAnnotation.id = 'lineAnnotation';
 const arrowHeadsDefaults = {
   backgroundColor: undefined,
   backgroundShadowColor: undefined,
+  borderCapStyle: 'butt',
   borderColor: undefined,
   borderDash: undefined,
   borderDashOffset: undefined,
+  borderJoinStyle: 'miter',
```

Comment:
Setting the default here, borderJoinStyle doesn't fallback to the line options.

https://github.com/chartjs/chartjs-plugin-annotation/blob/e94475cc299fd77dd6c008b7f4bfd4a985df3a6d/src/types/line.js#L198-L208